test/e2e/scheduling/priorities.go

```diff
@@ -28,9 +28,11 @@ import (
 	_ "github.com/stretchr/testify/assert"
 
 	v1 "k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Although it is not the main subject, some other files define the above line as `apierrors`.
For example, test/e2e/storage/subpath.go
It might be better to unify the definitions for readability.

Reply:
That is a nice point.
Actually we are managing this kind of thing with `./hack/.import_aliases` file and the above apierrs is not included at this time.
So it is nice to add it to the list.

I investigated current situation of `"k8s.io/apimachinery/pkg/api/errors"`.
Under `/test`, `apierrs` is more than `apierrors` as the following.
So it is fine to keep the above code today.
```
$ grep "k8s.io/apimachinery/pkg/api/errors" * -R | grep apierrs | wc
     30      90    2251
$ grep "k8s.io/apimachinery/pkg/api/errors" * -R | grep apierrors | wc
     14      42    1153
```